src/app.js

```diff
@@ -51,15 +52,15 @@ go.app = function() {
         });
 
         self.states.add('states:results', function(name, opts) {
-            var location_choices = _.map(opts.locations.slice(0, 2), function(d) {
+            var location_choices = _.map(opts.locations.slice(0,2), function(d) {
```

Comment:
Now that we're using the PaginatedChoiceState, we probably no longer need to slice the locations down to just 2. We can either just make choices for all of them, or limit them to a larger number -- maybe 10?